oceannavigator/frontend/src/components/AreaWindow.jsx

```diff
@@ -98,33 +108,6 @@ class AreaWindow extends React.Component {
     this._mounted = false;
   }
 
-  componentWillReceiveProps(props) {
```

Comment:
I assume this all gets to go away because we are now tracking some sort of global state?